crates/api/src/arc.rs

```diff
@@ -0,0 +1,305 @@
+//! Kitsune2 arcs represent a range of hash locations on the DHT.
```

Comment:
I was thinking that locations and arcs are not tied to hashes. Especially in this module, as is said in the following paragraph of the module docs, an arc can exist in its own right and just delimits a range of locations. Either way is fine with me tho.

Reply:
Right okay, yes that second paragraph is confusing. I've simplified that text because it wasn't really useful as it was. I did mean that it always refers to hash locations but it might mean "this time slice is for some DhtArc" or it might mean "this agent is an authority for this DhtArc". We can come back and add more documentation when this type is actually used that way and it can all be linked together properly